### DISCUSSION TITLE
I'm seeing 10TB more space than I expect in my raidz1 array

### DISCUSSION THREAD

**Discussion by komali2** [openzfs/zfs#14326] [Q&A] [answered]
I have a funky array of 14, 14, 18, 18 TB drives. I set them up as `raidz1`: 

```
sudo zpool create -f media raidz1 (drive 1, 2, 3, 4)
```

I got the error about drives being different size, and overroad it with `-f`. Now, I'm expecting at best 40TB of space, based on a bunch of random calculators I tried out (I mean in head math I don't see why I'd get more than 42 but this is black magic to me so whatever). Yet if I do `zpool list` , I see:

```
NAME     SIZE  ALLOC   FREE  CKPOINT  EXPANDSZ   FRAG    CAP  DEDUP    HEALTH  ALTROOT
media   50.9T   744K  50.9T        -         -     0%     0%  1.00x    ONLINE  -
```

That's only 8TB less than when I had this monster in a stripe. What gives?

**Comment by rincebrain** [comment, marked as answer]
`zpool list` includes the space occupied by parity overhead in its space, because (among other reasons) the ratio of parity overhead to space used isn't fixed.

`zfs list` (and df) give an estimate (because, again, things can be less efficient if you're storing smaller things) of space after that overhead.

You may find [this](https://www.delphix.com/blog/zfs-raidz-stripe-width-or-how-i-learned-stop-worrying-and-love-raidz) interesting reading.
